easybuild/easyconfigs/l/launcher/tests_modification.patch

```diff
@@ -0,0 +1,25 @@
+diff -ruN tests.orig/quicktest tests/quicktest
```

Comment:
Please add a short patch description on top, and include the author (and keep in mind that this means changing the checksum for this patch file accordingly)